src/renderer/Core/Settings/Pages/Appearance/Appearance.tsx

```diff
@@ -70,6 +71,7 @@ export const Appearance = () => {
         <SettingGroupList>
             <SearchBarSettings />
             <KeyboardShortcuts />
+            {contextBridge.getOperatingSystem() === "macOS" && <DockSettings />}
```

Comment:
These settings should not be under "Appearance" as the appearance settings are all settings on how the UI inside of the app looks. The dock settings are more related to how the app integrates with macOS. Therefore I suggest putting these settings to "General".